app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,9 @@
+%container
+  .row
+    .col-md-4.col-md-offset-4
+      .well
+        %h1.lead Sign In
+        = bootstrap_form_tag url: '/sign_in', method: 'POST' do |f|
```

Comment:
Instead of hardcoded url path, you should use named routes, more concise and if the stuff changes its easier to work with named routes than hardcoded content.